content/3_Silver/Binary_Search_Ans.mdx

```diff
@@ -536,6 +625,40 @@ static boolean check(long x){
 
 </JavaSection>
 
+<PySection>
+
+```python
+# checks whether the number of given operations is sufficient
+# to raise the median of the array to x
+def check(x):
+    operationsNeeded = 0
+	for i in range((n-1)//2,n):
+		operationsNeeded += max(0, x-v[i])
+    if operationsNeeded <= k:
+		return True
+    else:
+		return False
+
+
+# binary searches for the correct answer
+def search():
+    pos = 0
+	a = mx = 2E9
+	while a >= 1:
```

Comment:
and here

Reply:
This is really weird

Reply:
> and here

I'll try to fix this

Reply:
It looks like the problem is that you're using both tabs and spaces.